requirements.txt

```diff
@@ -17,3 +17,4 @@ tenacity
 antlr4-python3-runtime==4.11
 wikipedia
 wikipedia_sections
+vllm
```

Comment:
we should pin to a version. in my experiments 0.3.0 was good

Reply:
I implemented those changes but I had to rollback on that. VLLM has conflicting packages with prompting and bittensor, so if we pin a specific version the requirements won't work anymore. If we leave it open, vllm will be installed by pip trying to adjust the versioning, warning the conflicts rather than preventing the installation. 

```bash
The conflict is caused by:
    prompting 1.1.2 depends on transformers==4.36.2
    vllm 0.3.0 depends on transformers>=4.37.0
    bittensor 6.6.0 depends on pydantic!=1.8, !=1.8.1, <2.0.0 and >=1.7.4
    vllm 0.3.0 depends on pydantic>=2.0
```